pkgs/by-name/hy/HybridBar/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  glib,
+  pango,
+  cairo,
+  gtk3,
+  gtk-layer-shell,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "HybridBar";
+  version = "0.4.9";
+
+  src = fetchFromGitHub {
+    owner = "vars1ty";
+    repo = "HybridBar";
+    rev = "v${version}";
+    hash = "sha256-e9QVDDN8AtCZYuYqef1rzLJ0mklaKXzxgj+ZqGrSYEY=";
+  };
+
+  cargoHash = "sha256-50Go2AJQqgG2chc70KqKJexffD1rpObXr85ZPAIqc/U=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    gtk-layer-shell
+    gtk3
+    glib
+    pango
+    cairo
+  ];
+
+  meta = {
+    description = "A status bar focused on wlroots Wayland compositors";
```

Comment:
```suggestion
    description = "Status bar focused on wlroots Wayland compositors";
```

Reply:
Description taken from the HybridBar repo itself. 
Not changing

Reply:
Nonetheless articles at the beginning should be removed.